server/controllers/spreadsheet-controller.js

```diff
@@ -391,6 +392,15 @@ var errorList = {
     "pasting just your adjacency matrix into a fresh Excel Workbook and saving it."
   },
 
+  unknownFileError: {
+    errorCode: "unknownFileError",
+    possibleCause: "An unexpected error occured originating in the file.",
+    suggestedFix: "Please check the format of your spreadsheet with the guidlines outlined on the" +
```

Comment:
Whoops this actually was initial work for another issue I was working on before I took up fixing the SIF.